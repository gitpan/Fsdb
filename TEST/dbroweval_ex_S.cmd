prog='dbroweval'
args="-f TEST/dbroweval_ex.code"
cmp='diff -cb '
