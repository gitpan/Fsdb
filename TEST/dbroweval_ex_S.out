#fsdb -F S size mean stddev pct_rsd
1024   1496200    284970  19.047
10240   5028600    601030  11.952
102400   4921600    309390  6.2863
#  | dbsetheader size bw
#  | /home/johnh/BIN/DB/dbmultistats size bw
#  | /home/johnh/BIN/DB/dbcol size mean stddev pct_rsd
#   | dbroweval   { _mean = sprintf("%8.0f", _mean); _stddev = sprintf("%8.0f", _stddev);  }
